dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -570,7 +587,7 @@ def test_correct_total_sample_size_and_counts_and_mutability(self):
             self.assertEqual(0.5, profile._get_row_is_null_ratio())
             self.assertEqual(0.4375, profile._get_unique_row_ratio())
             self.assertEqual(9, profile._get_duplicate_row_count())
-        
+            
         self.assertEqual(col_one_len, len(data['NAME']))
```

Comment:
test pickling

Reply:
> Pickle files don't have a header, so there's no standard way of identifying them short of trying to unpickle one and seeing if any exceptions are raised while doing so.

https://stackoverflow.com/questions/13939913/how-to-test-if-a-file-has-been-created-by-pickle

Reply:
Removed, will add to a different PR